### ISSUE TITLE
Failure to create environment on AppVeyor with conda-forge

### ISSUE THREAD

**Issue by astrofrog** [conda-forge/conda-forge.github.io#208]
I'm seeing an error to do with duplicate files when creating an environment with the conda-forge channel:

```
[Error 183] Cannot create a file when that file already exists
```

Full log: https://gist.github.com/astrofrog/6798aafe1f4c71dfcb17e9658b7785de

**Comment by astrofrog** [comment]
I was accidentally using conda 4.1.3, but this is fixed in conda 4.1.11

**Comment by jakirkham** [comment]
Thanks for the info. Glad things are working for you now.
